packages/lit-dev-content/site/docs/v3/getting-started.md

```diff
@@ -55,7 +55,7 @@ standard JavaScript modules with no dependencies - any modern browser should be
 able to import and run the bundles from within a `<script type="module">` like this:
 
 ```js
-import {LitElement, html} from 'https://cdn.jsdelivr.net/gh/lit/dist@2/core/lit-core.min.js';
+import {LitElement, html} from 'https://cdn.jsdelivr.net/gh/lit/dist@3.0.0/core/lit-core.min.js';
```

Comment:
Do we maybe want to just specify major version here so users automatically get patches and minor non-breaking updates?
```suggestion
import {LitElement, html} from 'https://cdn.jsdelivr.net/gh/lit/dist@3/core/lit-core.min.js';
```